lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -294,26 +297,50 @@ if Enum.all?(
         manifest_module: Membrane.HTTPAdaptiveStream.HLS,
         target_window_duration: state.target_window_duration,
         target_segment_duration: state.target_segment_duration,
+        muxer_segment_duration: state.target_segment_duration - Membrane.Time.seconds(1),
         persist?: false,
         storage: %Membrane.HTTPAdaptiveStream.Storages.FileStorage{
           directory: directory
         },
         hls_mode: state.hls_mode
       }
 
+      audio_mixer = %Membrane.AudioMixer{
+        caps: %Membrane.RawAudio{
+          channels: 1,
+          sample_rate: 48_000,
+          sample_format: :s16le
+        }
+      }
+
+      {frames_per_second, 1} = state.compositor_config.output_framerate
+      seconds_number = div(state.target_segment_duration, Membrane.Time.seconds(1))
```

Comment:
Is this for avoid using Ratio?

We could just do `Membrane.Time.as_seconds(state.target_segment_duration)` but `Membrane.Time.as_seconds` can return ratio so we would have to handle it